.pipelines/build.yml

```diff
@@ -0,0 +1,35 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-2019'
+
+variables:
+  solution: 'JWT.sln'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+  dotNetVersion: '2.2.106'
+
+steps:
+- task: DotNetCoreInstaller@0
+  displayName: Install Dot Net Core v$(dotNetVersion)
+  inputs:
+      version: $(dotNetVersion)
+
+- task: MSBuild@1
+  displayName: Build $(solution)
+  inputs:
+    solution: $(solution)   
+    msbuildArguments: /restore /t:build /p:CreatePackage=true /p:NoPackageAnalysis=true /p:PackageOutputPath=$(Build.ArtifactStagingDirectory)\artifacts
+    configuration: $(BuildConfiguration)
+    maximumCpuCount: true
+
+- task: DotNetCoreCLI@2
+  displayName: Run Tests
+  inputs:
+    command: test
+    projects: '**/*Tests/*.csproj'
```

Comment:
Do you know why [the latest gated build](https://abatishchev.visualstudio.com/Jwt.Net/_build/results?buildId=83) finds no test projects?

Reply:
Tests are running now